### PR TITLE
Challenges of Advanced Algorithm Scripting translated.

### LINKED ISSUE #6719: Brazilian Portuguese translation of Advanced Algorithm Scripting
- [ ] Validate US Telephone Numbers
- [ ]  Symmetric Difference
- [ ]  Exact Change
- [ ]  Inventory Update
- [ ]  No repeats please
- [ ]  Friendly Date Ranges
- [ ] Make a Person
- [ ] Map the Debris
- [ ] Pairwise

### PR DESCRIPTION
**Pull request by kevinbg012**
Closes #6719

### CODE REVIEW

**Comment by BerkeleyFalse** [comment]
@kevao012 thanks for the PR.
Your branch name should start with one of `fix/, feature/` prefixes. Name, your branches correctly next time, please.
Do not include issue numbers and following [keywords](https://help.github.com/articles/closing-issues-via-commit-messages/#keywords-for-closing-issues) in commit messages.
Please, review our [**Guidelines for Contributing**](https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/CONTRIBUTING.md), thank you!.

**Comment by BerkeleyFalse** [comment]
@kevao012 updated the pull request.

File: seed/challenges/01-front-end-development-certification/advanced-bonfires.json [hunk 1]
```diff
@@ -75,6 +75,19 @@
         "<code>1 555 555 5555</code>",
         "Para esta prueba se te presentará una cadena de texto como por ejemplo: <code>800-692-7753</code> o <code>8oo-six427676;laskdjf</code>. Tu trabajo consiste en validar o rechazar el número telefónico tomando como base cualquier combinación de los formatos anteriormente presentados. El código de área es requrido. Si el código de país es provisto, debes confirmar que este es <code>1</code>. La función debe devolver true si la cadena de texto es un número telefónico válido en los EEUU; de lo contrario, debe devolver false.",
         "Recuerda utilizar <a href='//github.com/FreeCodeCamp/freecodecamp/wiki/How-to-get-help-when-you-get-stuck' target='_blank'>Read-Search-Ask</a> si te sientes atascado. Intenta programar en pareja. Escribe tu propio código."
+      ],
+      "namePtBR": "Validar Números Telefônicos dos EUA",
+      "descriptionPtBR": [
```
Comment (BerkeleyTrue) [inline seed/challenges/01-front-end-development-certification/advanced-bonfires.json]: Please change "namePtBR" to "titlePtBR"

Comment (kevao012) [inline seed/challenges/01-front-end-development-certification/advanced-bonfires.json]: Updated

**Comment by BerkeleyTrue** [comment]
@kevao012 Please fix your commit history. There should only be one commit for this PR.

**Comment by BerkeleyTrue** [comment]
Closing as stale. Feel free to re-open when ready.

**Pull request closed without merging**